src/platform/shared.c

```diff
@@ -538,6 +512,34 @@ void platform_process_messages()
 				break;
 			}
 			break;
+// Apple sends touchscreen events for trackpads, so ignore these events on OS X
+#ifndef __MACOSX__
```

Comment:
This macro is defined by SDL2. Since we are using SDL code in this file, I thought that it would make sense to just use it. I want to avoid accidentally excluding the code from any future iOS build, so it makes sense to do it this way.